core/src/Kokkos_Setup_HIP.hpp

```diff
@@ -0,0 +1,25 @@
+
+#if defined(KOKKOS_ENABLE_HIP)
+
+#define KOKKOS_IMPL_HIP_CLANG_WORKAROUND
+
+#define HIP_ENABLE_PRINTF
+#include <hip/hip_runtime.h>
+#include <hip/hip_runtime_api.h>
+
+#define KOKKOS_LAMBDA [=] __host__ __device__
+#endif  // #if defined(KOKKOS_ENABLE_HIP)
+
+#if defined(KOKKOS_ENABLE_HIP)
```

Comment:
```suggestion
```

Reply:
this has been fixed, but note that the file is now in core/src/setup/Kokkos_Setup_HIP.hpp